### DISCUSSION TITLE
希望增加MC服务器状态查询功能！

### DISCUSSION THREAD

**Discussion by AMGYzhea** [FloatTech/ZeroBot-Plugin#678] [Ideas]
根据IP查询对应的服务器status

**Comment by StuGRua** [comment]
简单做了一个这样的，看看能不能提上日程
![预览](https://github.com/FloatTech/ZeroBot-Plugin/assets/42519315/9facf831-54fc-42e2-8877-c4276e978a14)

**Comment by StuGRua** [comment]
See PR https://github.com/FloatTech/ZeroBot-Plugin/pull/1135
